com.github.maoschanz.DynamicWallpaperEditor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "com.github.maoschanz.DynamicWallpaperEditor",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dynamic-wallpaper-editor",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home:rw"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "dynamic-wallpaper-editor",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--libdir=lib"
+            ],
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/maoschanz/dynamic-wallpaper-editor.git"
```

Comment:
This has to be a stable, versioned, release.

Reply:
"versioned" means i have to write somewhere it's the 1.7, or just copy-pasting a commit id is fine enough ?

Reply:
Yes add:

```json
"tag": "1.7",
"commit": "9d9fda424acf22b012ba77b37971ab1716bcf8de"
```